### ISSUE TITLE
Gradle task jibDockerBuild fails with dockerClient config

### ISSUE THREAD

**Issue by ph-l** [GoogleContainerTools/jib#2172]
**Environment**:

- *Jib version:* v1.8.0-gradle
- *Build tool:* Gradle 5.6
- *OS:* Ubuntu 19.04

**Description of the issue**:
Using the `multi-module` example, providing a `dockerClient` configuration to the `jibDockerBuild` task returns an error:
```
gradle jibDockerBuild -Djib.dockerClient.executable=/usr/bin/docker

> Task :hello-service:jibDockerBuild FAILED
'jibDockerBuild.dockerClient.executable' is deprecated; use 'jib.dockerClient.executable' instead.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':hello-service:jibDockerBuild'.
> Cannot configure 'jibDockerBuild.dockerClient' and 'jib.dockerClient' simultaneously

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 621ms
2 actionable tasks: 2 executed
```

**Expected behavior**:
The task should succeed.

**Steps to reproduce**:

In `jib/examples/multi-module`, run `gradle jibDockerBuild -Djib.dockerClient.executable=/usr/bin/docker`

**`jib-gradle-plugin` Configuration**: 
```groovy
buildscript {
  repositories {
    mavenCentral()
    maven {
      url 'https://plugins.gradle.org/m2/'
    }
  }
  dependencies {
    classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.0.3.RELEASE'
    classpath 'io.spring.gradle:dependency-management-plugin:1.0.6.RELEASE'
    classpath 'gradle.plugin.com.google.cloud.tools:jib-gradle-plugin:1.8.0'
  }
}

subprojects {
  apply plugin: 'java'
  apply plugin: 'eclipse'
  apply plugin: 'idea'
  apply plugin: 'org.springframework.boot'
  apply plugin: 'io.spring.dependency-management'
  apply plugin: 'com.google.cloud.tools.jib'

  repositories {
    mavenCentral()
  }

  sourceCompatibility = 1.8
  targetCompatibility = 1.8

  dependencies {
    compile 'org.springframework.boot:spring-boot-starter-web'
  }

  // IMPORTANT: Set the environment variable PROJECT_ID to your own Google Cloud Platform project.
  jib.to.image = "gcr.io/${System.getenv('PROJECT_ID')}/${project.name}:${version}"
}
```

**Comment by TadCordle** [comment]
Thanks for reporting this, I was able to reproduce the problem. However I think the problem will be fixed in the next release, as we've removed the deprecated `jibDockerBuild.dockerClient` property.

If you'd like, you can try installing from master and using `com.google.cloud.tools:jib-gradle-plugin:1.8.1-SNAPSHOT` in your `build.gradle` (you'll also probably need to add `mavenLocal()` to your `repositories` block under `buildscript`) to see if it works.

**Comment by TadCordle** [comment]
Closing for now, if you run into more problems feel free to re-open with your findings.

**Comment by chanseokoh** [comment]
> Cannot configure 'jibDockerBuild.dockerClient' and 'jib.dockerClient' simultaneously

@TadCordle so is it that this message is wrong? For Jib 1.8 and lower, it's not possible to use `jib.dockerClient` and the user should use the deprecated `jibDockerBuild.dockerClient`?

**Comment by TadCordle** [comment]
@chanseokoh I think it's just the system property, actually. `jibDockerBuild.dockerClient` and `jib.dockerClient` both resolve the same `jib.dockerClient.executable` and `jib.dockerClient.environment` system properties, so if you use one, it assumes the docker client is configured on both the task and the extension.

**Comment by TadCordle** [comment]
@ph-l We've released 2.0.0, which removes the deprecated parameter and fixes this issue.
